.eslintrc

```diff
@@ -1,6 +1,14 @@
 {
-  "extends": ["./node_modules/react-scripts/node_modules/eslint-config-react-app/index.js"],
```

Comment:
yeah i don't know the best way to fix this. either `SKIP_PREFLIGHT_CHECK` or this i think are the only two options. I wasn't able to get resolutions to work properly

Reply:
Decided to go with `SKIP_PREFLIGHT_CHECK`. Seems to be working, for now.